lib/token-verifier/WP_Auth0_AsymmetricVerifier.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
[See this diff](https://github.com/auth0/auth0-PHP/pull/401/files#diff-71973553f78976e53afc2cfa2b20d861) for changes from approved PHP SDK ones.